lib/plugins/create/templates/aws-nodejs-ecma-script/serverless.yml

```diff
@@ -0,0 +1,26 @@
+service: babel-dynamically-entries-example
+
+# Add the serverless-webpack plugin
+plugins:
+  - serverless-webpack
+
+provider:
+  name: aws
+  runtime: nodejs6.10
+
+functions:
+# Example without LAMBDA-PROXY integration
+# Invoking locally:
+# sls webpack invoke -f first
```

Comment:
This sould be changed to `serverless invoke local` as soon as serverless-webpack v3 has been released, together with the package.json entry. Best would be in a follow up PR at a later time. For now it's ok.